python-checks/src/test/resources/checks/numpyWhereOneCondition.py

```diff
@@ -0,0 +1,47 @@
+def import_1():
+    import numpy as np
+    arr = np.array([1, 2, 3, 4])
+
+    empty = np.where()
+    args = (arr > 1, arr + 1, arr)
+    result = np.where(*args)
+    indices = np.nonzero(arr > 1)
```

Comment:
I think this test `indices = np.nonzero(arr > 1)` repeats itself quite a few times. Having it only once should be enough.